src/content/docs/developer-tools/kinde-api/access-token-for-api.mdx

```diff
@@ -112,9 +101,9 @@ If you decide to use Postman, we recommended that you set up a Postman **enviro
 />
 
 13. Select **Use Token**. You should now have the access token for making requests to the Kinde management API. See the [Kinde API documentation](https://kinde.com/api/docs/#kinde-management-api) for all the available end points.
-14. Next (optional): [Test the connection to Kinde’s API](/developer-tools/kinde-api/test-the-connection-to-kindes-api/).
+14. Next (optional): [Test the connection to Kinde’s API](/developer-tools/kinde-api/troubleshoot-kinde-api/).
 
-## Get the access token (Node.js fetch example)
+## Get the access token (Node.js fetch)
```

Comment:
Seems like we have a specific Kinde API x NodeJS doc, so possibly this should live there instead?

Reply:
I might look at this in a separate PR - will leave for now